packages/python/javdb_migrations/tools/reconcile_d1_drift.py

```
@@ -0,0 +1,1011 @@
+            logger.warning(
+                "%s: batched SELECT (%d keys) failed, falling back per-row: %s",
+                label, len(chunk_keys), batch_exc,
+            )
+            cursors = []
+            for k in chunk_keys:
+                try:
+                    cursors.append(d1.execute(sql, list(k)))
+                except D1Error as exc:
+                    logger.warning("%s: SELECT key=%s failed: %s", label, k, exc)
+                    cursors.append(None)
+        for key, cur in zip(chunk_keys, cursors):
```

Ruff (0.15.11)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)